Application/EdFi.Ods.Api/NHibernate/Composites/HqlBuilder.cs

```diff
@@ -404,29 +383,8 @@ public HqlBuilderContext CreateFlattenedMemberContext(HqlBuilderContext builderC
                         (char) (processorContext.ChildIndex + 'a'));
                 });
 
-            IEnumerable<EntityProperty> orderByProperties;
-
-            if (processorContext.ShouldUseHierarchy()
-                && processorContext.CurrentResourceClass.Entity.HasSelfReferencingAssociations)
-            {
-                var selfReferencingAssociation =
-                    processorContext.CurrentResourceClass.Entity.OutgoingAssociations
-                                    .Single(x => x.IsSelfReferencing);
-
-                var safelySortableProperties =
-                    selfReferencingAssociation.ThisProperties
-                                              .Intersect(selfReferencingAssociation.OtherProperties);
-
-                orderByProperties = locallyDefinedIdentifyingProperties
-                   .Intersect(safelySortableProperties);
-            }
-            else
-            {
-                orderByProperties = locallyDefinedIdentifyingProperties;
-            }
-
             // Add ORDER BY for the primary keys
-            orderByProperties.ForEach(
+            locallyDefinedIdentifyingProperties.ForEach(
```

Comment:
Is there a change for the approval tests for composites? If so did you deleted the saved file?

Reply:
This was a refactoring where code was removed that was no longer needed, but functionality and output should not have been affected for the non-hierarchical use cases. In re-reviewing the code, I see no approval tests for the _hierarchical_ SpecFlow feature.